recipes/elfio/all/test_package/example.cpp

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
```suggestion
```

Reply:
I don not know what this does, remove an empty line, so the next editor config will add it back?